src/@ionic-native/plugins/jins-meme/index.ts

```diff
@@ -0,0 +1,169 @@
+import { Cordova, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+declare var cordova: any;
+
+/**
+ * @name JinsMeme
+ * @description
+ * Implementation of the JINS MEME SDK
+ *
+ * @usage
+ * ```
+ * import { JinsMeme } from 'ionic-native';
+ *
+ * JinsMeme.functionName('Hello', 123)
+ *   .then((something: any) => doSomething(something))
+ *   .catch((error: any) => console.log(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'JinsMeme',
+  plugin: 'JinsMemeSDK-Plugin-Cordova', // npm package name, example: cordova-plugin-camera
+  pluginRef: 'cordova.plugins.JinsMemePlugin', // the variable reference to call the plugin, example: navigator.geolocation
+  repo: 'https://github.com/jins-meme/JinsMemeSDK-Plugin-Cordova.git', // the github repository URL for the plugin
+  install: '' // OPTIONAL install command, in case the plugin requires variables
+})
+@Injectable()
+export class JinsMeme {
+
+  /**
+  * Authentication and authorization of App and SDK.
+  * Must call this method at first.
+  *
+  *@param {string} setAppClientID
+  *@param {string} clientSecret
+  *@returns {Promise<any>}
+  */
+  @Cordova()
+  static setAppClientID(appClientId: string, clientSecret: string): Promise<any> { return; }
```

Comment:
All methods should not be static. Please remove the `static` keyword from all methods below.